src/main/scala/tile/BaseTile.scala

```diff
@@ -264,10 +264,6 @@ abstract class BaseTileModuleImp[+L <: BaseTile](val outer: L) extends LazyModul
 
   outer.traceAuxDefaultNode.bundle.stall := false.B
   outer.traceAuxDefaultNode.bundle.enable := false.B
```

Comment:
With the new default wire value for BundleBridgeNexus, it seems like the traceAuxDefaultNode is no longer needed.  Its purpose was to provide a default value in case there were no other input edges (from trace encoders).

Reply:
Correct! But you do have to supply the default value to the nexus. For e.g. `UInt` nexuses doing so is quite pithy, but for the aux node nexus I needed
```
val traceAuxNode = BundleBridgeNexus[TraceAux](
  default = Some({
    val aux = Wire(new TraceAux)
    aux. stall := false.B
    aux. enable := false.B
  })
)
```
and I wasn't sure that that was better than leaving the existing code as is since it is actually equal or more lines of code (modulo some futzing with whitespace). Is having the extra node onerous or does it help explain where the value is coming from when e.g. looking at the graphml. I could be convinced either way

Reply:
Understood.  I don't really have an opinion either way, so leaving it as is seems fine to me.

Reply:
i later decided to go with using the default because those wires became the only ones left in the `BaseTileModuleImp` class